src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        Ball chosenOne = new Ball();
+        chosenOne.setColor(getRandomColor());
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
+        chosenOne.setNumber(randomNumber);
+        return chosenOne;

```

Comment:
Temporary variable 'chosenOne' is not necessary. You can directly return the new `Ball` object without assigning it to a local variable.